modules/filetype.nix

```
@@ -17,7 +17,20 @@ let
       (listOf (
         either str (submodule {
           options = {
-            priority = mkOption { type = int; };
+            priority = mkOption {
+              type = ints.unsigned;
+              description = ''
+                Filename patterns can specify an optional priority to resolve cases when a file path
+                matches multiple patterns.
+
+                Higher priorities are matched first.
+                When omitted, the priority defaults to 0.
+
+                A pattern can contain environment variables of the form "${SOME_VAR}" that will be
```

```suggestion
                A pattern can contain environment variables of the form `"''${SOME_VAR}"` that will be
```
This will be interpolated as a nix expression without escaping the `$` as `''$`. Not sure why CI isn't complaining though, am I missing something?

Also, I'd add backticks, personally.